### PR TITLE
Fix image zooming accessibility issue

### PR DESCRIPTION
**Pull request by wandeg**
Issue: When the page is reflowed(Zoom to 400%), then content and images are overlapping in the community section​

### CODE REVIEW

**Comment by odero** [comment]
@wandeg Got a screenshot of the before and after?

File: pages/contribution.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 class="text-center" >Contribute to OData.org</h2>
 
 
 <h2 class="text-center">Participate elsewhere</h2>
-<div class="row" style="height:210px;text-align:center">
+<div class="row" style="height:auto;text-align:center">
```
Comment (odero) [inline pages/contribution.html]:
```suggestion
<div class="row text-center">
```

File: pages/contribution.html [hunk 2]
```diff
@@ -29,7 +29,7 @@ <h2 class="text-center">Participate elsewhere</h2>
 </div>
 
 <h2 class="text-center">Join discussion groups</h2>
-<div class="row" style="height:210px; text-align:center">
+<div class="row" style="height:auto; text-align:center">
```
Comment (odero) [inline pages/contribution.html]:
```suggestion
<div class="row text-center">
```

File: pages/contribution.html [hunk 3]
```diff
@@ -42,7 +42,7 @@ <h2 class="text-center">Join discussion groups</h2>
 </div>			
 
 <h2 class="text-center">Contact us</h2>
-<div class="row" style="height:210px; text-align:center">
+<div class="row" style="height:auto; text-align:center">
```
Comment (odero) [inline pages/contribution.html]:
```suggestion
<div class="row text-center">
```

**Comment by wandeg** [comment]
before icon
after icon
Uploaded before and after screenshots

**Comment by wandeg** [comment]
before icon
before

**Comment by wandeg** [comment]
after icon
after

File: _includes/footer.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿<footer class="content-info" role="contentinfo">
 
 <div class="container" style=" text-align: center">
-  <p>&copy; 2015-2018 OData – The Protocol for REST APIs </p>
+  <p>&copy; 2015-{{ 'now' | date: "%Y" }} OData – The Protocol for REST APIs </p>
```
Comment (marabooy) [inline _includes/footer.html]: The assumption here is the site is updated at least yearly. Could be improved by having this calculation client side

**Pull request merged**